timeutil/schedule.go

```diff
@@ -557,13 +682,20 @@ func parseWeekSpan(s string) (span WeekSpan, err error) {
 		parsed.End = parsed.Start
 	}
 
-	if parsed.End.Pos < parsed.Start.Pos {
-		// eg. mon4-mon1
-		return span, fmt.Errorf("cannot parse %q: unsupported schedule", s)
-	}
+	if (parsed.Start.Pos != EveryWeek) == (parsed.End.Pos != EveryWeek) {
+		// both ends have a week position set
```

Comment:
The above condition is also true if both don't have week position set at the same time, no? Should it say `if (parsed.Start.Pos != EveryWeek) && (parsed.End.Pos != EveryWeek)`?

Reply:
Thanks for the change; it also means we were probably missing a test for such case?

Reply:
No, it's already handled. Start.Pos == EveryWeek && End.Pos.EveryWeek happens for non numbered schedules, eg. mon-fri, (in which case Pos == 0 and Weekday is set), but those are indifferent for the later checks. Then, there's already a number of parser checks for numbered schedules, suchs as mon1-tue, mon-tue1, or mon1-tue2 (handled by the extra deprecated syntax check).